### ISSUE TITLE
Question/Feature Request? Project w. fixed rate per month...

### ISSUE THREAD

**Issue by VOOM108** [kimai/kimai#1167]
Don't know where else to ask. If to use the invoices with templates, I nee to be able to add a fixed rate per month. E.g. a customer needs to pay 200€ / month fixed rate and for extra custom work additionally times with an hourly fee. Is there a way to do that now already or could that be included in a future release?

**Comment by simonschaufi** [comment]
Hi, the only thing that works right now out of the box is that you assign a fixed rate on a task and just add a task every month. everything else will be per hour. you could even create a cronjob that does that automatically.

**Comment by VOOM108** [comment]
Great, thanks, that helps... Did not think of that. Adding the task per month just before exporting invoices will suffice :)

**Comment by VOOM108** [comment]
There is one more thing about the invoices - some templates have a <RN> placeholder for the invoice number. Is there a way to input that within kimai to produce a complete invoice or do I need to edit the output manually to include the invoice number? (Rechnungsnummer)

**Comment by simonschaufi** [comment]
Great. If you like the format of the number, you can use it of course but my own format is different so I always create it in my odt template manually. It's up to you.

**Comment by VOOM108** [comment]
It is not filled at all, so I don't even know, how the format would be. But I have a unique system to form the number and thus will probably have to edit the odt as well.

**Comment by simonschaufi** [comment]
The correct variable name is  `invoiceId`.

**Comment by VOOM108** [comment]
It's driving me crazy! The template I made was just ready, then suddenly it does not open in LibreOffice with an error: "Format error discovered in the file in sub-document content.xml at 2,32501(row,col)." Even though Explorer even generates a thumbnail. In MS Office (2007) same error (File damaged). I started from scratch and copied just the text to a new ODT with no effect. Now even invoices generated with the template 2016.odt I used as a base produce that error! I cannot seem to make a template with the content I need. All I used is text and tables.
